### PR TITLE
New Python versions in tests

### PR DESCRIPTION
**Pull request by frenzymadness**
I've used this as an exercise to understand the tests and now I'm gonna try to rewrite them for tox 4.

### CODE REVIEW

**Review by hroncok — CHANGES_REQUESTED** [review]
Could you please change https://github.com/fedora-python/tox-current-env#tests and https://github.com/fedora-python/tox-current-env/blob/master/setup.py#L36 as well?

**Comment by frenzymadness** [comment]
> Could you please change https://github.com/fedora-python/tox-current-env#tests and https://github.com/fedora-python/tox-current-env/blob/master/setup.py#L36 as well?

Done.

I'm gonna limit this PR to Python 3.10 only. My plan is to redefine the tests in #42 so adding the next Python will be much easier.

**Comment by frenzymadness** [comment]
All green, finally!

**Review by hroncok — APPROVED** [review]

File: .github/workflows/main.yaml [hunk 1]
```diff
@@ -21,20 +21,22 @@ jobs:
...
+        - py37-toxtox315
+        - py38-toxrelease
         - py38-toxmaster
+        - py38-toxtox315
+        - py39-toxrelease
         - py39-toxmaster
+        - py39-toxtox315
+        - py310-toxrelease
+        - py310-toxmaster
+        - py310-toxtox315
```
Comment (hroncok) [inline .github/workflows/main.yaml:39]: I have not spotted that this is a bug in tox configuration. Now we don't test with tox 3.15 at all.  Fix included in https://github.com/fedora-python/tox-current-env/pull/47

**Pull request merged**